common/vm_enable_nested_virtual.yml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2022 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Enable or disable nested virtualization on VM
+# Parameters:
+#   vm_nested_virt: true or false.
+# if set to true, will expose full CPU virtualization to the guest operating system
+#
+- name: "Check parameter"
+  ansible.builtin.assert:
+    that:
+      - vm_nested_virt is defined
+      - vm_nested_virt | type_debug == 'bool'
+    fail_msg: "Parameter vm_nested_virt is required and set to be a boolean."
+
+- name: Enable VM CPU Virtualization
```

Comment:
If this task file can be used to enable or disable nested virtualization, it is not proper to use `vm_enable_nested_virtual.yml` as the file name. You can change it to `vm_set_nested_virtual.yml` or something like that. And in line 16 and line 29, you can't only mention `enable` as it could be `disable`.